atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -237,6 +238,15 @@ public int getDefaultLockTimeoutSeconds() {
         return AtlasDbConstants.DEFAULT_LOCK_TIMEOUT_SECONDS;
     }
 
+    /**
+     * The max time to wait for a thread when concurrently running range requests. Note this should
+     * never really be hit, as the underlying KVS should have a much tighter timeout.
+     */
+    @Value.Default
+    public Duration getConcurrentGetRangesTimeout() {
```

Comment:
I have no idea how Immutables / Jackson magic deal with specifying Duration objects in lieu of the primitives that all of the other configurables are using

Reply:
Looks like not well -- tests currently failing on this. Guess will just use a string and convert.